includes/forms/template.php

```diff
@@ -486,6 +486,12 @@ function give_output_levels( $form_id ) {
 	$display_style      = give_get_meta( $form_id, '_give_display_style', true );
 	$custom_amount      = give_get_meta( $form_id, '_give_custom_amount', true );
 	$custom_amount_text = give_get_meta( $form_id, '_give_custom_amount_text', true );
+	$minimum_amount     = give_get_meta( $form_id, '_give_custom_amount_minimum', true, 'custom' );
```

Comment:
@raftaar1191 is `custom` is default custom amount? Please check core before setting a default value.